### PR TITLE
Sorting Throttling Tiers by reverse order

### PR DESCRIPTION
**Pull request by lahiru823**
Please commit the changes

### CODE REVIEW

File: modules/apps/publisher/themes/appm/js/resource-add.js [hunk 1]
```diff
@@ -250,6 +250,7 @@ function updateUserRoles(index) {
  */
 function drawThrottlingTiersDynamically() {
     var strContent = "";
+    tiers.reverse();
```
Comment (rushmin) [inline modules/apps/publisher/themes/appm/js/resource-add.js]: Why the tiers list is reversed ?

File: modules/apps/publisher/themes/appm/helpers/edit-asset.js [hunk 2]
```diff
@@ -57,7 +57,9 @@ var extractVisibleRoles = function(data){
 		}
 
 		// Or else check against the exclude list.
-		var excludedRolesList = ["admin", "subscriber", "publisher"];
+		var config = require("/config/publisher.json");
+		var excludedRolesList = config.excludedRolesList;
+
```
Comment (dinusha-dilrukshi) [inline modules/apps/publisher/themes/appm/helpers/edit-asset.js]: Remove multi line spaces.

File: modules/apps/publisher/themes/appm/js/resource-add.js (hunk 1 above)
Comment (lahiru823) [inline modules/apps/publisher/themes/appm/js/resource-add.js]: Revered the order to set Unlimited as the 1st (default)

File: modules/apps/publisher/themes/appm/helpers/edit-asset.js (hunk 2 above)
Comment (lahiru823) [inline modules/apps/publisher/themes/appm/helpers/edit-asset.js]: fixed with commit# https://github.com/lahiru823/carbon-appmgt/commit/1d831effdd9484a443781b45dfe757d7e6558266

**Pull request merged**